src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -183,7 +183,7 @@ CorProfiler::Initialize(IUnknown* cor_profiler_info_unknown) {
     }
   }
   if (FAILED(hr)) {
-    Warn("Failed to attach profiler: unable to set event mask.");
+    Warn("DATADOG TRACER DIAGNOSTICS - Failed to attach profiler: unable to set event mask.");
```

Comment:
I'm not sure what qualifies for `DATADOG TRACER DIAGNOSTICS`, but do the above domain-neutral logs need this message prefix or no?

Reply:
As I understand it, `DATADOG TRACER DIAGNOSTICS` should be used for stuff that causes the instrumentation to fail. The domain-neutral logs does not necessarily indicate an error condition.